views/templates/details.jade

```diff
@@ -10,7 +10,8 @@ mixin renderConnection(conn)
         = " "
         = conn.contact_person
       if conn.phone && (tAttr(conn.name).indexOf(conn.phone) === -1)
-        = ", "
+        if (tAttr(conn.name)).length > 1
```

Comment:
What if the name is `I` or `U` or some other one letter name? 😉

Reply:
Oops, fixed 😄